scpd/lms/static/sass/_mobile.scss

```diff
@@ -0,0 +1,567 @@
+/****************************************************************************************************
+*   Mobile Theming File
```

Comment:
Do we still need this, in light of changes upstream?

Reply:
Yes we still do, because we currently don't have all of their mobile changes, and we've customized quite a bit of stuff.

Reply:
Maybe, but this should stay until we have the merge for testing. Removing it is super easy, delete this file and the include in `_overrides`.